rabbitmq-server-3.13.yaml

```diff
@@ -21,17 +21,20 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - elixir
+      - elixir-1.15
```

Comment:
is there no -dev package? and is it linked dynamically? in most cases that is fine to use like that. Or we ressurect that version stream of elixir in wolfi / or move this rabbitmq to enterprise packages if we still need to ship it.

Reply:
setup v1.15 in wolfi (following past conventions and no guidance to do otherwise)